internal/db/dao.go

```diff
@@ -82,6 +83,27 @@ func (dao *PostgresDAO) GetGoals(pagination Pagination, filter GetGoalsFilter) (
 	return list, nil
 }
 
+func (dao *PostgresDAO) GetGoal(id string) (top90.Goal, error) {
+
+	query := fmt.Sprintf("SELECT * FROM %s WHERE id is %s", tableNames.Goals, id)
+
+	var list top90.Goal
```

Comment:
Don't need this variable. We can just move the `var video top90.Goal` up here and use it.